recipes/libs/fribidi.yaml

```diff
@@ -1,20 +1,21 @@
-inherit: [autotools]
+inherit: [meson]
 
 metaEnvironment:
-    PKG_VERSION: "1.0.10"
+    PKG_VERSION: "1.0.11"
 
 checkoutSCM:
     scm: url
     url: ${GITHUB_MIRROR}/fribidi/fribidi/releases/download/v${PKG_VERSION}/fribidi-${PKG_VERSION}.tar.xz
-    digestSHA256: 7f1c687c7831499bcacae5e8675945a39bacbad16ecaa945e9454a32df653c01
+    digestSHA256: 30f93e9c63ee627d1a2cedcf59ac34d45bf30240982f99e44c6e015466b4e73d
     stripComponents: 1
 
+buildTools: [host-toolchain]
```

Comment:
I'm unsure about this one and there's probably a better solution to it, but without this line configure fails with

```
configure: error: cannot find native C compiler
```

Reply:
this happens if configure searchs explicitly for "gcc" or "g++".
normally the target-toolchain compiler is named like: x86_64-cross-linux-gnu-gcc...

Reply:
The target toolchain is found, actually, but the package needs the host toolchain, too (for tools or tests or IDK)...

Reply:
Any further input on this? I actually have a second package (`libs::freetype`) needing the exact same "fix" and I'd like to bring my fork back on par with upstream...

Reply:
Well if the package needs the host toolchain then I see no problem in using it. Otherwise we would need to patch the package but I don't see a benefit in doing that...